From 28817688fc563879a8731893e7360795c3b830e7 Mon Sep 17 00:00:00 2001
From: Leed Aguilar <leed.aguilar@ti.com>
Date: Mon, 28 Feb 2011 12:25:26 -0600
Subject: OMAP4: Increase Kernel log buffer size

Increase Kernel log buffer size by 16KiB to
allow debugging the complete Kernel boot up
information.

Recently many Kernel Subsystems have added
several debbuging information at boot time
and the previous Kernel buffer size was not
big enough to store all that data and many
important boot up information was overwritten

Change-Id: I9e193d047343e4a8d5fd2ad50801decc7d5171dd
Signed-off-by: Leed Aguilar <leed.aguilar@ti.com>
---
 arch/arm/configs/android_4430_defconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

Index: kernel/arch/arm/configs/android_4430_defconfig
===================================================================
--- kernel.orig/arch/arm/configs/android_4430_defconfig	2011-04-02 13:12:43.000000000 +0900
+++ kernel/arch/arm/configs/android_4430_defconfig	2011-04-02 13:14:11.000000000 +0900
@@ -67,7 +67,7 @@
 # CONFIG_RCU_FAST_NO_HZ is not set
 # CONFIG_TREE_RCU_TRACE is not set
 # CONFIG_IKCONFIG is not set
-CONFIG_LOG_BUF_SHIFT=14
+CONFIG_LOG_BUF_SHIFT=15
 # CONFIG_CGROUPS is not set
 # CONFIG_SYSFS_DEPRECATED_V2 is not set
 # CONFIG_RELAY is not set
